distribution/pom.xml

```diff
@@ -390,71 +390,71 @@
                   <directory>/opt/drill</directory>
                   <sources>
                     <source>
-                      <location>target/apache-drill-${project.version}/apache-drill-${project.version}/git.properties</location>
+                      <location>target/${project.parent.artifactId}-${project.version}/${project.parent.artifactId}-${project.version}/git.properties</location>
```

Comment:
all the `<location>target/...` can be replaced by `<location>${project.build.directory}/....`

Reply:
Good suggestion. Done.